modoboa_radicale/forms.py

```diff
@@ -6,6 +6,12 @@
 from modoboa.lib import form_utils
 from modoboa.parameters import forms as param_forms
 
+SSL_VERIFY_CHOICES = (
+    (True, ugettext_lazy("Verify")),
+    (False, ugettext_lazy("Skip verification")),
+    ("path", ugettext_lazy("Provide CA bundle"))
+    )
```

Comment:
Bad indentation and I don't think it's a good idea to mix types here. Use either strings or integers for values.

Reply:
Like that ? 
I don't see the bad indent though

Reply:
Yes, that's better.
And the last parenthesis should be moved to start of line

Reply:
done :D